### DISCUSSION TITLE
Workshop Dependencies3 is missing content

### DISCUSSION THREAD

**Discussion by BertDeTemmerman** [microsoft/AL-Go#1570] [Ideas]
The [Dependencies3](https://github.com/microsoft/AL-Go/blob/d9d3b069e8e3d8891a3602a75e3261d85d8f4555/Workshop/Dependencies3.md) workshop doesn't has much content.
Can/will this be updated on how to do this?
The [AL-Go for GitHub: Add a dependency to a private 3rd party app](https://www.youtube.com/watch?v=a06eetoPaAU&list=PLpD0iWf1mwXTH5V4Oc3TcyYE8a7zb-EvC&index=22) youtube video is quite brief.

**Comment by jonaswre** [comment]
Not sure what the plan for that part of the documentation. But the how to is quite brief. 

Download the app from a partner portal upload them to a safe blob storage and include the URL to the App file in the following property in your AL-Go settings. ```
"installApps": ["URL to your App File"] ``` 

That's it.

**Comment by BertDeTemmerman** [comment]
Ah great, this is already something.
(For a newbie like me, I already got a clearer view what the URL represents.)

But that's just that version you will be using.

You don't get the latest/release version.
In order to get acces to that version from github directly, the 3th party has to open the artifacts for download, and use that URL in `installApps`?



The Dependencies3 also has 4 titles but no content for each one.

**Comment by freddydk** [comment]
Will get to work on the workshop later this year (hopefully)
